service_container.rst

```diff
@@ -255,71 +349,91 @@ Next, tell the container the service has a constructor argument:
                 http://symfony.com/schema/dic/services/services-1.0.xsd">
 
             <services>
-                <service id="app.message_generator" class="AppBundle\Service\MessageGenerator">
-                    <argument type="service" id="logger" />
-                </service>
+                <!-- ... -->
+
+                <!-- Registers all classes in Services & Updates directories -->
+                <prototype namespace="AppBundle\" resource="../../src/AppBundle/{Service,Updates,EventDispatcher,Twig,Form}" />
```

Comment:
shouldn't we add all directories supported in the standard edition (`{Command,Form,EventSubscriber,Twig,Security}`)?